packages/theme/css/foundations/fade.css

```diff
@@ -49,6 +49,7 @@
   --salt-color-gray-600-fade-background-readonly: rgba(47, 49, 54, var(--salt-palette-opacity-background-readonly));
   --salt-color-gray-800-fade-background-readonly: rgba(36, 37, 38, var(--salt-palette-opacity-background-readonly));
 
+  --salt-color-white-fade-backdrop: rgba(255, 255, 255, var(--salt-palette-opacity-backdrop));
   --salt-color-black-fade-backdrop: rgba(36, 37, 38, var(--salt-palette-opacity-backdrop));
```

Comment:
Should this be black? I can't find info about "fade" in the theme characteristics Figma, but it is used here`--salt-palette-neutral-backdrop: var(--salt-color-black-fade-backdrop);` and in the Figma it looks like `--salt-palette-neutral-backdrop` should be `color-black, opacity 70%`.

cc @feliciackh

Reply:
You should follow Figma

Reply:
@lilyvc good catch - i've message Paul to enquire further into specific values for said token. In the Theme updates excel sheet, line 63, as well as this [gh issue](https://github.com/jpmorganchase/salt-ds/issues/2190), noted the change from using `color-black` to `color-white` so that was intended. Looks like we never had `var(--salt-color-white-fade-backdrop)` defined to begin with, let's see what Paul says

Reply:
Just to confirm I've replied to Felicia directly. `white-fade` should be #FFFFF and `black-fade` should be #000000, currently its incorrectly set to the equivalent of grey-800.